### PR TITLE
Added a larger amount of missing brands in Sweden

### PR DESCRIPTION
**Pull request by matthiasfeist**

### CODE REVIEW

File: data/brands/amenity/cafe.json [hunk 1]
```diff
@@ -3663,6 +3663,19 @@
...
       }
     },
+    {
+      "displayName": "Thelins konditori",
+      "id": "thelinskonditori-46f791",
+      "locationSet": {"include": ["se"]},
+      "tags": {
+        "amenity": "cafe",
+        "brand": "Thelins konditori",
+        "brand:wikidata": "Q124048792",
```
Comment (LaoshuBaby) [inline data/brands/amenity/cafe.json:3673]: It only has 18 branches shop. Does it really satisfy the notability?

Comment (matthiasfeist) [reply data/brands/amenity/cafe.json:3673]: Yeah that's a good question. I've seen some other smaller chains in the index before, but I'm also fine with not adding it :)

Comment (bhousel) [reply data/brands/amenity/cafe.json:3673]: I think 18 is ok as a number, but the first thing I'm going to look at is Q124048792 and right now it looks pretty empty or spammy.  I worry that Wikidata admins might remove it if you don't add some supporting details from other sources.

Comment (matthiasfeist) [reply data/brands/amenity/cafe.json:3673]: that's all the data I have for them right now. What do you think would be missing?

**Comment by Snowysauce** [comment]
The Wikidata item for Holy Greens was deleted; see #10303

**Pull request merged**